src/index.js

```diff
@@ -230,6 +230,23 @@ class App extends React.Component {
             const y = e.clientY - e.target.offsetTop;
             const element = newElement(this.state.elementType, x, y);
 
+            let isDraggingElements = false;
+            const cursorStyle = document.documentElement.style.cursor;
+            if (this.state.elementType === "selection") {
+              isDraggingElements = elements.some(el => {
```

Comment:
I’ve been using the local variable “element” instead of “el”, we should try to be consistent everywhere.